Lib/test/test_typing.py

```diff
@@ -3738,6 +3738,18 @@ def test_repr(self):
 
         self.assertEqual(repr(UserId), f'{__name__}.UserId')
 
+    def test_pickle(self):
+        UserId = NewType('UserId', int)
+
+        s = pickle.dumps(UserId)
```

Comment:
Please test for all supported protocols.

Reply:
For Yurii: there's a good example in `test_pickletools` https://github.com/python/cpython/blob/main/Lib/test/test_pickletools.py#L24. It might also be better to add in a ``self.subTest`` in that loop.

Reply:
@Fidget-Spinner Thanks for a great example)